tests/cases/fourslash/server/projectInfo.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference path="../fourslash.ts"/>
+
+// @Filename: a.ts
```

Comment:
I would add another check for b.ts, b should only have b.ts and lib.d.ts; also i would add a test for ///references, and maybe a file that does not have any references, so it should be an island by itself.